client/blocks/login/login-form.jsx

```diff
@@ -360,23 +359,6 @@ export class LoginForm extends Component {
 		}
 	}
 
-	renderLoginFromSignupNotice() {
```

Comment:
Moving this notice outside of the form so we can render it just below the page heading.